### DISCUSSION TITLE
Question about the "direct_url.json" file

### DISCUSSION THREAD

**Discussion by Claudia-Hello** [janosh/matbench-discovery#76] [Q&A] [answered]
Hi all,
I have run this repository, but there was no "direct_url.json" file in the "matbench-discovery" folder. Could you please tell me how to solve this problem? The source coda can be seen as follows. Thank you so much!

pkg_name = "matbench-discovery"
direct_url = Distribution.from_name(pkg_name).read_text("direct_url.json") or ""
pkg_is_editable = json.loads(direct_url).get("dir_info", {}).get("editable", False)

**Comment by janosh** [comment, marked as answer]
thanks for reporting! converted this to issue #77. will be fixed in #78 and released as v1.1.1.
